src/main/java/io/cdap/plugin/gcp/publisher/source/GoogleSubscriber.java

```diff
@@ -118,9 +118,9 @@ private ZonedDateTime getTimestamp(String publishTime) {
    */
   public static class SubscriberConfig extends GCPReferenceSourceConfig {
 
-    @Description("Cloud Pub/Sub subscription to read from. If the subscription does not exist it will be " +
-      "automatically created if a topic is specified. Messages published before the subscription was created will " +
-      "not be read.")
+    @Description("Cloud Pub/Sub subscription to read from. If a subscription with the configured name does not " +
+      "exist it will be automatically created if a topic is specified. Messages published before the subscription " +
```

Comment:
We use 'configured' and 'specified' here, as well as 'provided' and 'given' in the topic description. Let's use a single word. My feeling is that 'specified' is most common in our descriptions.

Let's also fix the grammar and consistency of the topic description. How about we have the following for subscription:

Cloud Pub/Sub subscription to read from. If the specified subscription does not exist, it will automatically be created. Messages published before the subscription was created will not be read.

and for the topic:

Cloud Pub/Sub topic to subscribe to. If the specified topic does not exist, it will automatically be created. (remove the note, as that belongs in the subscription description)

Reply:
I think the description for topic you suggested implies that the topic would be created, instead of a subscription to that topic.

PTAL

Reply:
OK this thing is super confusing. Was wondering why the topic was nullable, and it's because the topic is only used when the subscription needs to be auto-created and is otherwise never used. How about:

Cloud Pub/Sub topic to subscribe to. This is only used when the specified subscription does not already exist and needs to be automatically created. If the specified subscription already exists, this value is ignored.

Reply:
sgtm